### ISSUE TITLE
What about a log of all games?

### ISSUE THREAD

**Issue by TheTechRobo** [mmatyas/supermariowar#201]
Sometimes I want to look back at my first few games, but I can't since I didn't record them. Could we have like a log file?

**Comment by mmatyas** [comment]
Hi, you mean something like a replay file? I fear that would require larger changes to the game code, but that would be indeed a nice feature.

**Comment by TheTechRobo** [comment]
Yeah, sort of, but even just "who won, who lost, how many points did everyone have in the end, mode settings, etc" would be an amazing start. :D
